### ISSUE TITLE
Vasco applies wrong shield modifier

### ISSUE THREAD

**Issue by Constellation-VASCO** [Starfield-Community-Patch/Starfield-Community-Patch#648]
Vasco has Shield Systems level 2 but when assigned to a ship only gives a 20% bonus. The skill description says that level 2 should add 40% shield strength. So either the modifier is wrong or Vasco should only have Shield Systems level 1 (which matches the 20%) 

---

## Details 
Technically when Vasco is assigned the `ShieldMaxHealthMult` modifier is increased by 0.2 and is decreased again by 0.2 when Vasco is unassigned. That value should be 0.4 instead. My guess is that Bethesda buffed the skill but forgot to correct Vasco's numbers.

## Game Version
1.7.36.0

## Language
English (EN)

## New Game
Yes

## Reproduction Steps
* Assign Vasco to the ship. Shield goes up by 20% but should go up 40%

**Comment by Constellation-VASCO** [comment]
Just found https://www.reddit.com/r/Starfield/comments/16f2kx2/guide_crew_skills_explained_and_optimal_crew/ which shows that the crew skill levels have nothing to do with the player skill levels... So level 2 in Shield Systems brings 20% while level 3 brings 20% plus a bonus shield pip. I guess it makes sense because another 60% bonus when hiring Omar would be overkill... To bad all this info is not displayed directly in the game.

I think the bug can be closes because it is not a bug (or the bug is that the game doesn't explain how all this works). Sorry for the inconvenience.

**Comment by ja231c** [comment]
I'd say this is a bug as it does not work in a way that a reasonable person would expect it to work nor is there any explanation as to why it works the way it works.

**Comment by ArkangelCorvus** [comment]
I agree this should not be considered a bug. The problem here is lack of documentation ingame, since (most of) the crew perk functionality is explicitly described in the perk descriptions, easily readable in xEdit:
![image](https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/assets/7544079/6a5fea2a-387d-49c5-ab5f-8d3b5af5b33c)
I think this makes the developers' intentions quite clear. Displaying these descriptions in the interface would be nice too, but out of scope for the project.
